docs/default.nix

```diff
@@ -40,8 +40,8 @@ let
         patches = old.patches or [ ] ++ [
           # Adds support for GFM-style admonitions in rendered commonmark
           ./0001-Output-GFM-admonition.patch
-          # TODO:add support for _parsing_ GFM admonitions too
-          # https://github.com/nix-community/nixvim/issues/2217
+          # Adds support for parsing GFM-style admonitions
+          ./0001-nixos-render-docs-Support-gfm-style-admonitions.patch
```

Comment:
Nit: ideally the patch should be generated on top of the other patch, and then this would be `0002`